### PR TITLE
Add Bahmni: Standalone Document Upload Module

### PR DESCRIPTION
**Pull request by alpit4**
## Bahmni 

Standalone Document Upload Module

### Changes

What was updated (new pages, weekly update, etc.)

### Notes

If there are any further notes.

### CODE REVIEW

**Comment by GeekGawd** [comment]
Please remove package-lock.json and yarn.lock

**Comment by alpit4** [comment]
I have deleted the yarn.lock and package.json file. Now,please review and merge it.

**Review by shubhigupta991 — APPROVED** [review]

File: docs/2023/Bahmni/Standalone Document Upload Module/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+# Author
+Alpit Kumar
+
+# Contact Info
+[Email](mailto:alpitkumar85@gmail.com) 
+[Linked In](https://www.linkedin.com/in/alpit-kumar-301265224/) 
+[GitHub](https://github.com/alpit4)
```
Comment (claddyk) [inline docs/2023/Bahmni/Standalone Document Upload Module/index.md:14]:
```suggestion
- [Email](mailto:alpitkumar85@gmail.com) 
- [Linked In](https://www.linkedin.com/in/alpit-kumar-301265224/) 
- [GitHub](https://github.com/alpit4) 
```

**Review by claddyk — COMMENTED** [review]
You should not delete the yarn.lock in your commit.

**Comment by alpit4** [comment]
Now What I can do.

**Pull request closed without merging**